view/frontend/web/js/view/payment/method-renderer/adyen-pm-method.js

```diff
@@ -211,7 +211,11 @@ define(
                             self.isPlaceOrderAllowed(state.isValid);
                         },
                     });
-                return configuration
+
+
+                configuration = Object.assign(configuration, paymentMethodsExtraInfo[paymentMethod.type].configuration);
+
+                return configuration;
```

Comment:
`paymentMethodsExtraInfo` has been injected in `adyen-cashapp-method.js` file. Is it still required here?